ci-operator/jobs/red-hat-storage/odf-console/red-hat-storage-odf-console-release-4.11-presubmits.yaml

```diff
@@ -0,0 +1,182 @@
+presubmits:
+  red-hat-storage/odf-console:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - ^release-4\.11$
+    - ^release-4\.11-
```

Comment:
Compatibility jobs should to target 4.12.

Reply:
This PR only enables for 4.11 branch. The rehearses are only ran for 4.11 branch

Reply:
Oh, that's cool then! 👍🏼